arelle/ModelXbrl.py

```diff
@@ -22,35 +22,34 @@
 
 if TYPE_CHECKING:
     from arelle.ModelRelationshipSet import ModelRelationshipSet as ModelRelationshipSetClass
-    from arelle.ModelInstanceObject import ModelContext
+    from arelle.ModelInstanceObject import ModelContext, ModelFact, ModelDimension, ModelUnit
+    from arelle.ModelValue import QName, DATETIME
+    from datetime import date, datetime
+    from arelle import ModelFormulaObject, ModelManager
+
 
 
 ModelFact = None
 
-profileStatNumber = 0
+profileStatNumber: int = 0
```

Comment:
These literal values don't need to be typed. mypy can infer `x = 0` is an int or `x = 'foo'` is a string, etc.